src/Vlingo.Xoom/actors/Settings.cs

```diff
@@ -9,47 +9,54 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Reflection;
+using Vlingo.Cluster.Model;
 
-namespace io.vlingo.xoom.actors {
-    public class Settings {
-
-        private static Dictionary<string, string> PROPERTIES = new Dictionary<string, string>();
-        private static readonly string PROPERTIES_FILENAME = "/vlingo-xoom.properties";
-        private static readonly long serialVersionUID = 1L;
-        private static readonly Dictionary<object, object> DEFAULT_DATABASE_PROPERTIES = new Dictionary<object, object>() {
+namespace Vlingo.Xoom.Actors
+{
+    public class Settings
```

Comment:
Inherit from `ConfigurationPropeties` in `vlingo-net-custom` as discussed. As I already explained overs slack it loads JSON formatted properties

Reply:
Is it okay just inheriting from ConfigurationPropeties. Or do i need to make some changes also?